### PR TITLE
RD-6180 Remove passing `bordered` prop from icons

### PR DESCRIPTION
**Pull request by Vorbert-Kruk**
## Description

Changes consists of:
- Removing `bordered` prop from the `<Icon />` component
- Updating `cloudify-ui-components` 
- Providing translations for modified icons 🎉 

## Screenshots / Videos

I'm attaching screenshot from just one of the modified places, to show the direction in which we are heading 🥳 

### Before
Stage

### After
Stage

## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4066/)

As yesterdays nightly job was aborted, I've manually executed [System Tests on the master branch](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/4065/), so that I could compare it with the previously executed job to see if anything would break after delivering my changes.

As it turns out, 1 system test was "extra" failing - [flows/user_spec.ts](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/test/cypress/integration/flows/user_spec.ts)
But the cause of it seems not to be related to my changes - after clicking "create" button in `Providers Managament` widget, a corresponding modal is not appearing

As I was suspecting an issue related to the pipeline performance, I've executed another [System Test only related to the user_spec.ts](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/4072) - it's passing 🎉 

So with all that being said, I don't think that my changes are breaking anything 💪  

## Documentation

N/A

### CODE REVIEW

File: app/translations/en.json [hunk 1]
```diff
@@ -1905,6 +1909,26 @@
...
+                "download": "Download",
+                "delete": "Delete"
+            }
+        },
+        "managers": {
+            "actions": {
+                "openConsole": "Open Console",
+                "refreshStatus": "Refresh Status",
+                "refreshStatusInProgress": "Status refresh in progress..."
+            }
```
Comment (qooban) [inline app/translations/en.json:1931]: Great 🎉

**Review by qooban — APPROVED** [review]
Looks good 👍🏻

**Comment by qooban** [comment]
> As it turns out, 1 system test was "extra" failing - [flows/user_spec.ts](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/test/cypress/integration/flows/user_spec.ts)
> But the cause of it seems not to be related to my changes - after clicking "create" button in Providers Managament widget, a corresponding modal is not appearing

Good catch. We'll need to spread that knowledge, so:
@Blewin Be aware of that issue (see the error description above) when you'll be creating corresponding ticket(s).
@PatrycjaKubiczek I guess you'll need to take care about it.

**Comment by Vorbert-Kruk** [comment]
@qooban 
> Good catch. We'll need to spread that knowledge

Yeah, agreed
But when it comes to actions that we can take, I'd wait and see if there will be other similar failures, as it could be just a one timer related to the jenkins stability ([next execution of that test](https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/4072) was a success)
If the issue wouldn't be reappearing, I don't think that there would be a need from @PatrycjaKubiczek side to to anything 😁 

WDYT? 🤔

**Comment by qooban** [comment]
@Vorbert-Kruk I suspects it's necessary to take action anyway. In [test/cypress/integration/flows/user_spec.ts](https://github.com/cloudify-cosmo/cloudify-stage/blob/d504f3c95d879f95c6be281d5295f86c3071eadb/test/cypress/integration/flows/user_spec.ts) we were assuming there's just one widget on Secrets page. Since #2303 we have two widgets there - Secrets widget and Secret Providers widget. Both have "Create" buttons, so we need to be more specific when it comes to clicking it 🙂 Patrycja is on vacation today, but I believe she can easily fix it within #2315 (or as part of separate PR) in the beginning of the next week.

**Comment by Vorbert-Kruk** [comment]
> @Vorbert-Kruk I suspects it's necessary to take action anyway. In [test/cypress/integration/flows/user_spec.ts](https://github.com/cloudify-cosmo/cloudify-stage/blob/d504f3c95d879f95c6be281d5295f86c3071eadb/test/cypress/integration/flows/user_spec.ts) we were assuming there's just one widget on Secrets page. Since #2303 we have two widgets there - Secrets widget and Secret Providers widget. Both have "Create" buttons, so we need to be more specific when it comes to clicking it 🙂 Patrycja is on vacation today, but I believe she can easily fix it within #2315 (or as part of separate PR) in the beginning of the next week.

@qooban oh, in that case it totally makes sense 🎉 
Thanks sharing more details about the situation 💪

**Comment by PatrycjaKubiczek** [comment]
@qooban @Vorbert-Kruk implemented changes mentioned in this PR to my PR :) https://github.com/cloudify-cosmo/cloudify-stage/pull/2315/commits/735fe606aad128e3c68e192c1a45bc2e8a738d47 thanks for noticing it! 🙌 👍

**Pull request merged**